app/src/main/java/com/pusher/demo/features/marketplace/ChatkitManager.kt

```diff
@@ -75,4 +108,18 @@ object ChatkitManager {
             }
         )
     }
+
+    fun getAllUsersInitialState(listener: ChatkitManager.ChatManagerAllUsersListener) {
```

Comment:
as this is something specific to the seller, wouldn't it be better to keep it inside its specific code?

Reply:
Makes sense - it cleans a bunch of the other things you've left comments on too :-)